okteto.yml

```diff
@@ -0,0 +1,18 @@
+name: hello-world
+build:
+  hello-world:
+    context: .
```

Comment:
let's add the `image` field as we did here:
https://github.com/okteto/go-getting-started/pull/10/files#diff-dd8070292ce806a326a3d0543b136ba3b5f7930f40d33475780462fdcece2b70R3

I would use a different name, for example `okteto.dev/node-hello-world:1.0.0`.

This is the image we should use in the k8s deployment manifest, like here:
https://github.com/okteto/go-getting-started/pull/10/files#diff-9aa5c3faef5c8a780db30dc0dd41b946e5732af5677ee2fc17d3db774fe1a57bR16